build.sh

```diff
@@ -1,13 +1,13 @@
 #!/usr/bin/env bash
 
-set -euo pipefail
+set -eo pipefail
```

Comment:
Why drop `-u` here?

Reply:
running `build.sh` without providing `OPENSSL_VERSION` gives a cryptic error.

I prefer the script notice it's missing and give a human appropriate error.